lib/Attribute.js

```diff
@@ -205,16 +205,16 @@ Attribute.prototype.setTypeFromRawValue = function(value) {
     typeVal = value.type;
   }
 
-  if (util.isArray(typeVal) || typeVal === 'list'){
-    type = 'List';
-  } else if ( (util.isArray(typeVal) && typeVal.length === 1) || typeof typeVal === 'function') {
+  if ( (util.isArray(typeVal) && typeVal.length === 1) || typeof typeVal === 'function') {
```

Comment:
Will this cause merge conflicts with #521? Maybe we should ensure that we merge that PR in first, or?

Reply:
Yes it probably will cause a conflict. I think when you merge this into master you should be able to reconcile the differences?